release-notes/unreleased/IGAPP-1048-show-not-found-error.yml

```
@@ -0,0 +1,6 @@
+issue_key: IGAPP-1048
+show_in_stores: true
+platforms:
+  - web
+en: Show Not Found Error instead of Response Error on invalid endpoints
```

Perhaps:
```suggestion
en: Show correct error message for non-existing categories
```
Not Found Error, Response Error and endpoints are just implementation details and not really relevant for non-devs.